docs/FAQ.md

```diff
@@ -1,20 +1,17 @@
 # Frequently Asked Questions
 
-* TOC
-{:toc}
```

Comment:
Please restore the Table of Contents for the website.

Reply:
ToC is restored.